### DISCUSSION TITLE
🐱‍💻申請github education有多殺!付費資源通通免費~拿到pro級標籤

### DISCUSSION THREAD

**Discussion by JunTingLin** [1101-GS4719-C/Python-Community-GS4719-C#415] [General]
# 🐱‍💻申請github education有多殺!付費資源通通免費~拿到pro級標籤
![螢幕擷取畫面 2022-03-08 135513](https://user-images.githubusercontent.com/92431095/157183600-f0c4d93d-bb1d-453c-a0f0-863bf1e4f621.png)

## github education有很香嗎❓
那麼大家肯定很好奇，GitHub Education 提供學生那些優惠好康呢？最重要當然是 GitHub 的 Business Plan 和 GitHub Enterprise 存取權限，而且享有無限制的 private repositories（售價是每月 $7 美元），其他還有像是 AWS、DigitalOcean 和 Namecheap 這類架站工具的額度和產品，對於我們學生是不可多得的免費資源。
![螢幕擷取畫面 2022-03-08 135631](https://user-images.githubusercontent.com/92431095/157183621-b914112d-bfc9-4a01-a5f9-7456fa5fd1ed.png)

## github學生禮包是大家都可以申請嗎?
對滴，現在只要有學校提供的 Email 信箱，或是上傳學生證用以證明學生身分，都可獲取 GitHub Education 資格😁

## 如何申請
1. [github education](https://education.github.com/)登入github帳號
2. 按照步驟指引，填入學生信箱`XXX@cc.ncu.edu.tw`，等收到確認信後就知道是中央大學的學生了
3. 告訴他你還在學，可以傳學生證或是在學證明

詳細圖文教學請看: [點我](https://free.com.tw/github-education/)

## 節語
據我所知可以在.TECH購買免費網域，也可以透過Microsoft Azure、DigitalOcean部署網站，而且你的github主頁面還會顯示pro等級的字樣，反正不拿白不拿，有興趣的可以申請看看🚩。
![螢幕擷取畫面 2022-03-08 145547](https://user-images.githubusercontent.com/92431095/157183639-1332cc68-5055-4dde-b784-22e0b56020f4.png)

## 參考
[GitHub Education 開放所有學生免費使用軟體開發套裝，學校信箱就可申請](https://free.com.tw/github-education/)

[中央大學google club社群](https://gdsc.community.dev/national-central-university/)

**Comment by ThomasMeng0515** [comment]
感謝分享，讓我們可以免費升級~
p.s.內文有些錯字哈哈哈

**Reply by JunTingLin** [reply]
😅感謝提醒，錯字已更正

**Comment by chia-kai-chang** [comment]
這邊也有教師的工具箱喔：https://education.github.com/toolbox

其實這個組織[NCU-GS4719-Python](https://github.com/NCU-GS4719-Python)也是Github贊助升級的~ 如果真的要付錢，我就算去餐廳洗碗也付不出那麼多錢~ 感恩讚嘆Github

![image](https://user-images.githubusercontent.com/1768891/157269151-efe319df-9d64-4157-8376-a925e4381770.png)

**Reply by chia-kai-chang** [reply]
畢竟Github背後是Microsoft富爸爸~ 有錢真好~ 有錢卡司好~

**Comment by pekoranoinu** [comment]
感恩樓主分享，免錢的東西最香了。

**Comment by bloodtrail0511** [comment]
wc太有料了，趕緊去申請，不用錢就是讚

**Comment by sean1325418** [comment]
趁現在還是學生身分，能辦就辦

**Comment by xkllkx** [comment]
謝謝樓主分享，白嫖真香~

**Comment by henrywen547** [comment]
謝謝樓主分享不用錢的東西~

**Comment by hapiwangy** [comment]
之前就有在其他地方看過相關資訊了，在看過這裏的介紹之後\才發現原來有這麼多的好處

**Comment by PoooYen** [comment]
實用好文感謝分享

**Comment by Yuhsin000** [comment]
香噴噴，有申請到就賺到!!

**Comment by Sazmafuji** [comment]
能夠用到免費pro版,
好賺

**Comment by SamTU0222** [comment]
感謝原po
讓我可以免費升級呀

**Comment by HuangHsinTzu** [comment]
感謝分享!!!
看到「免費」兩個字
眼睛瞬間為之一亮~~

**Comment by ruthchen1020** [comment]
微軟爸爸對學生真好，我也想去那裡工作

**Comment by StevenWanglolz** [comment]
感謝分享，我們全都可以升級成pro版的github了!

**Comment by KH1126** [comment]
免費pro版的github太香了，
我覺得github copilot蠻有用的，
雖然可能因為無法像chatgpt一樣問問題，
因此他所提的一些建議不是我要的，
但總體來說對寫程式提升效率有幫助。

**Comment by YifenLin** [comment]
感謝樓主分享，免費升級pro真的香。之前在滑ig 的時候有看到一篇文章https://www.instagram.com/p/C3xj2Wwy1w-/?igsh=MTRybXE0YnZpM3htbA==
![IMG_7886](https://github.com/NCU-GS4719-Python/Python-Community/assets/138841470/c677541c-ea91-4c83-8a0d-2b7323975e62)

我認為升級pro 版除了有更多功能能夠使用之外，也能讓我們更好的與業界接軌。

**Comment by William08290829** [comment]
感謝github讓學生有些功能可以使用，也感謝後面的富爸爸微軟真的是讚讚讚帥帥帥，雖然我還是覺得他們家的有些軟體真的很難用但還是感謝他們願意提供給學生有這麼好的環境使用。

**Comment by jim10212023** [comment]
感謝分享，當初只為了copilot去申請，沒想到除了copilot 外，還有這麼多以前不知道的好康福利，長知識了。
身為學生的福利真的好多，github教育版真的很讚，謝謝github願意提供這麼讚的優惠給學生，只恨沒有早點發現，晚了這麼多才體驗到，也謝謝原PO無私地分享。

**Comment by cassy0201** [comment]
謝謝分享~真的是看到賺到!

**Comment by NcuJerryKao** [comment]
GitHub Education提供的免費資源對於學生和教育機構來說確實是一個非常有價值的福利。擁有GitHub的Business Plan和GitHub Enterprise存取權限，以及無限制的私人存儲庫，對於學生來說是一個強大的工具，可以幫助他們在學習和專案開發中更好地管理程式碼和專案。
申請GitHub Education的過程相對簡單，只需要提供學校提供的電子郵件或上傳學生證明學生身分，就可以獲得相應的資格。這為學生提供了更多學習和實踐的機會，也讓他們能夠更輕鬆地學習和應用技術。
對於有興趣的同學來說，申請GitHub Education是一個不錯的選擇，可以充分利用這些免費資源來提升自己的技術能力和學習成果。希望這些資源能夠幫助更多學生在技術領域中取得成功，並為其未來的發展打下良好的基礎。
